dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunts/package.order

```diff
@@ -1,2 +1,3 @@
 ShuntBWithSections
 ShuntBWithSections_INIT
+ShuntB
```

Comment:
ShuntB could be the first one in the package.order because it is the simplest model

Reply:
done